pkg/blockatlas/lending.go

```diff
@@ -0,0 +1,71 @@
+package blockatlas
+
+type ProviderType string
+
+const (
+	ProviderTypeLending ProviderType = "lending"
+	ProviderTypeStaking ProviderType = "staking"
+)
+
+type (
+	// LendingProvider static info about the lending provider, such as name and asset classes supported.
+	LendingProvider struct {
+		ID     string              `json:"id"`
+		Info   LendingProviderInfo `json:"info"`
+		Type   ProviderType        `json:"type"`
+		Assets []AssetInfo         `json:"assets"`
+	}
+
+	// LendingProviderInfo basic information about a lending provider.
+	LendingProviderInfo struct {
+		ID          string `json:"id"`
+		Description string `json:"description"`
+		Image       string `json:"image"`
+		Website     string `json:"website"`
+	}
+
+	// AssetInfo Info about an asset that can be lent
+	AssetInfo struct {
+		Symbol         string        `json:"symbol"`
+		Description    string        `json:"description"`
+		APY            float64       `json:"apy"`
+		YieldPeriod    int64         `json:"yield_period"` // the period of validity of current APY, 0 for variable APY
```

Comment:
Why not create standard across staking models too? it's the same info

Reply:
I don't want to touch Staking API now in any way, it would have to be extended.  But it's possible to create new version of Staking API unified with Lending API.